resources/model/OpenStudio.idd

```diff
@@ -30404,6 +30404,151 @@ OS:Generator:FuelSupply,
+      \note If this field is blank, the system is always available.
+      \type object-list
+      \object-list ScheduleNames
+  A4, \field Rotor Type
+      \note allowed values are: Horizontal Axis Wind Turbine or Vertical Axis Wind Turbine
+      \type choice
+      \key HorizontalAxisWindTurbine
+      \key VerticalAxisWindTurbine
+      \required-field
+  A5, \field Power Control
+      \note Constant power output is obtained in the last three control types
+      \note when the wind speed exceeds the rated wind speed.
+      \note allowed values are: Fixed Speed Fixed Pitch, Fixed Speed Variable Pitch,
+      \note Variable Speed Fixed Pitch or Variable Speed Variable Pitch
+      \type choice
+      \key FixedSpeedFixedPitch
+      \key FixedSpeedVariablePitch
+      \key VariableSpeedFixedPitch
+      \key VariableSpeedVariablePitch
+      \required-field
+  N1, \field Rated Rotor Speed
+      \required-field
+      \type real
+      \minimum> 0.0
+      \units rev/min
+  N2, \field Rotor Diameter
+      \note This field is the diameter of the perpendicular circle of the Vertical Axis Wind Turbine system
+      \note from the upright pole on the ground.
+      \required-field
+      \type real
+      \minimum> 0.0
+      \units m
+  N3, \field Overall Height
+      \note This field is the height of the hub for the Horizontal Axis Wind Turbines and
+      \note of the pole for the Vertical Axis Wind Turbines.
+      \required-field
+      \type real
+      \minimum> 0.0
+      \units m
+  N4, \field Number of Blades
+      \minimum 2
+      \required-field
+  N5, \field Rated Power
+      \required-field
+      \note This field is the nominal power at the rated wind speed.
+      \note Users should input maximum power in case of Fixed Speed Fixed Pitch control type.
+      \type real
+      \minimum> 0.0
+      \units W
+  N6, \field Rated Wind Speed
+      \required-field
+      \type real
+      \minimum> 0.0
+      \units m/s
+  N7, \field Cut In Wind Speed
+      \required-field
+      \type real
+      \minimum> 0.0
+      \units m/s
+  N8, \field Cut Out Wind Speed
+      \required-field
+      \type real
+      \minimum> 0.0
+      \units m/s
+  N9, \field Fraction system Efficiency
+      \type real
+      \minimum> 0.0
+      \maximum 1.0
+      \required-field
+  N10,\field Maximum Tip Speed Ratio
+      \type real
+      \minimum> 0.0
+      \maximum 12.0
+      \required-field
+  N11,\field Maximum Power Coefficient
+      \note This field should be input if the rotor type is Horizontal Axis Wind Turbine
+      \type real
+      \minimum> 0.0
+      \maximum 0.59
+      \required-field
+  N12,\field Annual Local Average Wind Speed
+      \type real
+      \units m/s
+      \minimum> 0.0
+  N13,\field Height for Local Average Wind Speed
+      \type real
+      \units m
+      \minimum> 0.0
+      \required-field
+  N14,\field Blade Chord Area
+      \type real
+      \units m2
```

Comment:
```suggestion
  N14,\field Blade Chord Area
      \type real
      \units m2
      \required-field
      \minimum 0.0
```

Same as Blad Draft Coef and Lift Coef

Reply:
E+ code has checks to see if VAWT is used while these fields are zero: https://github.com/NREL/EnergyPlus/blob/d6cf92a2a7dc8ca3489d39792c4315f6f3bea957/src/EnergyPlus/WindTurbine.cc#L515-L562

Reply:
Should it be `\minimum> 0.0` then?

Reply:
The idf in testfiles has blanks for the three Blade fields...

Reply:
I know, since the field isn't used if horizontal, but that doesn't mean it hurts any to set it to zero by default.